pkg/tsdb/testdatasource/testdata/labels.csv

```diff
@@ -0,0 +1,3 @@
+"a{foo=bar,x=y}",b{foo=baz},"{foo=x, no=name}"
```

Comment:
aww, it needs to be wrapped in quotes :(

can we do something else to avoid it? like a slightly more complex parser or

```
a{foo=bar;x=y},b{foo=baz},{foo=x; no=name}
```